pkg/plumber/plumber.go

```diff
@@ -66,23 +71,31 @@ func (b *PipelineBuilder) Create(request *PipelineOptions) (*PipelineOptions, er
 	}))
 	l.Info("about to start Jenkinx X meta pipeline")
 
-	po := create.StepCreatePipelineOptions{
-		SourceURL: sourceURL,
-		Job:       job,
-		PullRefs:  pullRefs,
-		Context:   spec.Context,
-	}
 	sa := os.Getenv("JX_SERVICE_ACCOUNT")
 	if sa == "" {
 		sa = "tekton-bot"
 	}
-	po.CommonOptions = b.commonOptions
-	po.ServiceAccount = sa
 
-	err := po.Run()
+	pipelineCreateParam := metapipeline.PipelineCreateParam{
+		PullRef:      pullRefData,
+		PipelineKind: kind,
+		Context:      spec.Context,
+		// No equivalent to https://github.com/jenkins-x/jx/blob/bb59278c2707e0e99b3c24be926745c324824388/pkg/cmd/controller/pipeline/pipelinerunner_controller.go#L236
```

Comment:
Right, so I would think that there are a few pieces (labels, env variables) missing here to make sure that jobs triggered by Lighthouse behave the same as jobs triggered by Prow